cutils.h

```diff
@@ -299,7 +375,7 @@ static inline int dbuf_put_u64(DynBuf *s, uint64_t val)
     return dbuf_put(s, (uint8_t *)&val, 8);
 }
 int __attribute__((format(printf, 2, 3))) dbuf_printf(DynBuf *s,
```

Comment:
Does the function still need the `__atribute__` after the `FORMAT_STRING` you added?

Reply:
Yes, it still need, beacause in non MSVC, FORMAT_STRING is gone